aten/src/ATen/cuda/CUDAGraph.cu

```diff
@@ -0,0 +1,30 @@
+#include <ATen/cuda/CUDAGraph.h>
```

Comment:
Including this file in a .cu file causes some warnings that were normally silenced to appear (e.g., assigning -1 to CaptureId_t, which is  uint64_t). See if I can silence those or otherwise get rid of them.